File: data/python/PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES_MAIN.py
if __name__ == '__main__':
    param = [
    (46,),
    (68,),
    (4,),
    (12,),
    (56,),
    (14,),
    (81,),
    (29,),
    (26,),
    (40,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        try:
            if f_filled(*parameters_set) == f_gold(*parameters_set):
                print("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + sys.argv[1] + "," + str(i) + ",SUCCESS")
            else:
                print("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + sys.argv[1] + "," + str(i) + ",FAILURE")
        except:
            print("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(46);
    param0.add(68);
    param0.add(4);
    param0.add(12);
    param0.add(56);
    param0.add(14);
    param0.add(81);
    param0.add(29);
    param0.add(26);
    param0.add(40);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
	        {
                builder.append("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("PIZZA_CUT_PROBLEM_CIRCLE_DIVISION_LINES," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
